example/src/index.scss

```diff
@@ -1,4 +1,4 @@
 @import "@edx/brand-openedx/paragon/fonts";
 @import "@edx/brand-openedx/paragon/variables";
-@import "@edx/paragon/scss/core/core";
```

Comment:
Webpack resolver did not work for this because of the missing `~` at the start of the import, changing this line `~@edx/paragon/scss/core/core` makes example app correctly import `scss` module from local Paragon repo instead of its `node_modules`.